test/Forwarder.test.ts

```diff
@@ -81,6 +89,34 @@ contract('Forwarder', ([from]) => {
     })
   })
 
+  describe('#registerDomainSeparator', () => {
+    it('registered domain should match local definition', async () => {
```

Comment:
I find "it should " a better test name pattern as it describes the desired behaviour, not the outcome.
Honestly, this sentence just re-states what the asserts check, not what the method is doing.

Reply:
ok, suggest something else.

Reply:
Its a sanity check, that on-chain calculation matches host (local) calculation.

Reply:
at compares actual values that are hashed into the domain-separator, so its not has "hash is different" but actual data that gets into this hash